bundles/binding/org.openhab.binding.mios/src/main/java/org/openhab/binding/mios/internal/config/DeviceBindingConfig.java

```diff
@@ -241,7 +247,7 @@
 	}
 
 	private static final Pattern SERVICE_IN_PATTERN = Pattern
-			.compile("service/(?<serviceName>.+)/(?<serviceVar>.+)");
+			.compile("service/(?<serviceName>.+)(/(?<serviceVar>.+))*");
```

Comment:
You'll want to use a REGEX Pattern more like "_X_?" (_X_, once or not at all)  at the end instead of "_X_*" (_X_, zero or more times) so the pattern-match is as tight as possible.